monai/fl/client/monai_algo.py

```diff
@@ -80,55 +68,38 @@ def compute_weight_diff(global_weights, local_var_dict):
     return weight_diff
 
 
-def check_bundle_config(parser):
```

Comment:
I think we should consider backward compatibility here as well. If we keep bundle_root, then RequiredBundleKeys can also be checked by this function. It is used by NVFlare [here](https://github.com/NVIDIA/NVFlare/blob/2cd69e0401124d8c441f5c4d99b04e76d47038a9/integration/monai/monai_nvflare/monai_bundle_persistor.py#L19) for example.

Reply:
I already added the check function in `BundleWorkflow`, so all the model-zoo bundles must support the MONAI FL required properties:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/workflows.py#L129

Thanks.